### PR TITLE
TM-1111 turn off helpers on mobile device

### PR DESCRIPTION
**Pull request by ArekZc**

### CODE REVIEW

File: src/container-debugger/widgets/breakpoints/_breakpoints.html [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <div id="breakpoints">
     <div id="breakpoints-title">
         <%= __('Breakpoints') %>
-        <cc-help content="<%= __('Option to break forwarding of links. Important when debug external or internal links') %>" placement="up"></cc-help>
+        {{ isMobile() | json }}
```
Comment (marnits) [inline src/container-debugger/widgets/breakpoints/_breakpoints.html]: you should remove this

Comment (kallosz) [inline src/container-debugger/widgets/breakpoints/_breakpoints.html]: you should hide not remove helper.

File: src/container-debugger/app.js [hunk 2]
```diff
@@ -36,11 +36,12 @@ angular
 
             $rootScope.debugger = $debugger;
             $rootScope.$state = $state;
+
             $rootScope.isMobile = () => {
 
                 return $window.parent.screen.availWidth < 400 && $window.orientation === 0;
```
Comment (kallosz) [inline src/container-debugger/app.js]: `400` should be `CONST`

File: src/container-debugger/app.js [hunk 3]
```diff
@@ -36,11 +36,12 @@ angular
 
             $rootScope.debugger = $debugger;
             $rootScope.$state = $state;
+
             $rootScope.isMobile = () => {
```
Comment (kallosz) [inline src/container-debugger/app.js]: method is not tested.

**Pull request merged**